requirements.txt

```diff
@@ -2,6 +2,9 @@ fastapi
 uvicorn
 pyyaml
 requests
+elasticsearch[async]==8.6.1
+numpy
+pandas
```

Comment:
numpy and pandas are already in the requirements.txt below, so only adding elasticsearch is sufficient.